CDP.ts

```diff
@@ -158,9 +159,6 @@ export class CDP {
     }
 
     private getDomainDc({dataCenter, env} = this.options) {
-        if (dataCenter != 'il1' && env == 'prod')
-            return dataCenter;
-
```

Comment:
do not remove this - this is to make sure that when requesting a `prod` env, then it's being dropped from the domain. it's relevant for eu5 and us5

Reply:
you right , anyway i checked and the api's works on both domains ( cdp.eu5.gigya.com and  cdp.eu5-prod.gigya.com- is it ok? )
so @zoe , pls keep it as it was.